engine/src/main/java/org/terasology/rendering/dag/RenderPipeline.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2016 MovingBlocks
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.terasology.rendering.dag;
+
+import java.util.ArrayList;
+
+/**
+ *
+ */
+public class RenderPipeline extends ArrayList<Object> {
```

Comment:
Errr... below I only suggested to use the `optimizedPipeline.execute() `method. The RenderPipeline is an intermediate stage. I suspect however even the optimized pipeline class could be eliminated. We could just use a List<RenderTask> and iterate that somewhere.

I think the only reasons we might want to keep those lists wrapped in their own class is to add debug functionality, i.e. methods to print out the current contents of the lists. But we could to that elsewhere too. What do you think?